importers/wecdis_importer.py

```diff
@@ -209,11 +253,15 @@ def handle_position(self, data_store, line_number, tokens, datafile, change_id):
         combine_tokens(source_token, sensor_token).record(self.name, "sensor", sensor_name)
 
         platform = self.get_cached_platform(data_store, self.platform_name, change_id=change_id)
-        sensor = self.get_cached_sensor(
-            data_store=data_store,
-            sensor_name=sensor_name,
-            sensor_type=sensor_token.text,
-            platform_id=platform.platform_id,
+        sensor_type = data_store.add_to_sensor_types("Location-Satellite", change_id=change_id).name
```

Comment:
@robintw - we're processing a line representing ownship position here.  There are around  6 different ownship tracks in this format, each from a different sensor. Since the sensors are predictable, we don't ask the analyst to add/create them - we just add them.

So, `data_store.add_to_sensors` may get called for every position line.  (Finally...) here's my worry:  do we cache the sensor data?  Or do we  have the cost of checking if the sensor already exists in the database?

Reply:
We don't cache anything in `add_to_sensors`, no. That will do a db lookup for the information, and either return the existing entry or insert a new one if it doesn't already exist. Is there a reason you stopped using `get_cached_sensor`?

Reply:
>  Is there a reason you stopped using get_cached_sensor?

Yes, to remove the user having to answer/acknowledge the prompts. There are ~6 ownship tracks for each file of this data-type. The analyst isn't adding any value by viewing/acknowledging each new sensor. So, I suggested to @mew-nsc  that we just create the sensor for the source, and move on - without any user interaction.

This _looked-like_ the correct way to handle that logic.  Is there another way?

Note: I compared old & new WECDIS importer (#rows/sec). The old one (with the requirement to acknowledge each new sensor) had about the same bulk import performance as the new one - though the new one avoided the need to answer the ownship track sensor answers.

Reply:
Interesting that the speed was roughly the same. That may slow down with a remote Postgres server over a slow-ish network though.

I was about to tell you to use the `unknown=True` argument for the `get_cached_sensor` call, but that doesn't actually exist at the moment. That would cache the sensor, but also treat it as unknown and not ask the user any information about it. Probably best to create a separate issue to add that functionality to `get_cached_sensor` and then we can upgrade the importer to use that later.